front/src/applications/stdcm/components/StdcmWarningBox.tsx

```diff
@@ -32,6 +37,28 @@ const StdcmWarningBox = ({
   const hasMissingFields = (errorDetails?.missingFields?.length ?? 0) > 0;
   const hasRouteErrors = (errorDetails?.routeErrors?.length ?? 0) > 0;
 
+  const infraId = useSelector(getStdcmInfraID);
+  const { infra } = useInfraStatus({ infraId });
+  const { pathfinding, isPathFindingLoading: _ } = useStaticPathfinding(infra);
+  const isFailureWithIncompatibleConstraints =
```

Comment:
Maybe this could be enough
```suggestion
  const hasIncompatibleConstraints =
```

Reply:
Done